src/write_sensu.c

```diff
@@ -115,9 +115,7 @@ static int sensu_connect(struct sensu_host *host) /* {{{ */
 		host->res = NULL;
 		hints.ai_family = AF_INET;
 		hints.ai_socktype = SOCK_STREAM;
-#ifdef AI_ADDRCONFIG
-		hints.ai_flags |= AI_ADDRCONFIG;
-#endif
+		hints.ai_flags = AI_ADDRCONFIG;
```

Comment:
You rename the variable to `ai_hints` above. Could you unify this, i.e. change this here, too?